### ISSUE TITLE
Cassandra consistency TWO prevents Kong startup

### ISSUE THREAD

**Issue by Tigraine** [Mashape/kong#1913]
### Summary

Hi, I am running kong inside docker from the mashape/kong:cassandra3 image with a Cassandra 3.5 5 node cluster.
I have setup 5 cassandra nodes with the Kong replication_factor of 5 (so 100% replication).
By setting KONG_CASSANDRA_CONSISTENCY_LEVEL: TWO in my ENV variables kong will be unable to start.

I think I have tracked down the issue in the code base: 
https://github.com/Mashape/kong/blob/next/kong/dao/db/cassandra.lua#L521

Consistency-level of `TWO` **will prevent ALL queries to the system, system_schema table** as it is is replicating with `LocalStrategy` and will never run with more than 1 alive replica. Setting the consistency to `ONE` or `QUORUM` makes kong start up again as that's something the LocalStrategy can serve - but not TWO as that's by design preventing any reads from `system_schema`.

A solution here would be to run that specific query with a different consistency level than what is configured. (Unfortunately I am terrible at lua so no PR)

I did try this out in CQLSH though to verify that my cluster is responding to consistency TWO. The kong keyspace does everything from ONE, TWO, THREE, QUORUM, LOCAL_QUORUM, ALL without problems. I believe it must be the system tables here.

Just out of curiosity - what's the suggested consistency level for a very high load application that needs maximum availability. I had it at ONE first but while testing cassandra failure scenarios sometimes underreplicated nodes would miss a row in kong.apis so I decided I need at least TWO to guard against new nodes coming up. (TWO obviously not working).
So now I am running Kong with QUORUM but that means I cannot loose more than 2 out of 5 nodes - whereas 3 out of 5 with TWO would have been better.

Thanks for the awesome product though - also Galileo - that thing saved us countless hours debugging issues so far..

### Steps To Reproduce

1. Cassandra cluster at least two nodes
2. Startup Kong with CONSISTENCY = TWO
3. Watch the fireworks

### Additional Details & Logs

- Kong version 0.9.0
```
$ kong start
Error: 
/usr/local/share/lua/5.1/kong/cmd/start.lua:15: [Unavailable exception] Cannot achieve consistency level TWO
stack traceback:
        [C]: in function 'assert'
        /usr/local/share/lua/5.1/kong/cmd/start.lua:15: in function 'cmd_exec'
        /usr/local/share/lua/5.1/kong/cmd/init.lua:81: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:81>
        [C]: in function 'xpcall'
        /usr/local/share/lua/5.1/kong/cmd/init.lua:81: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:42>
        /usr/local/bin/kong:11: in function 'file_gen'
        init_worker_by_lua:38: in function <init_worker_by_lua:36>
        [C]: in function 'pcall'
        init_worker_by_lua:45: in function <init_worker_by_lua:43>
```

**Comment by se** [comment]
Is it running on ONE ?

It is not running for me with 2 node of Cassandra and consistency = ONE.

Do you guys have any idea?

**Comment by Tigraine** [comment]
In my case it's running with `ONE` and `QUORUM`/`LOCAL_QUORUM`

it's just that by definition a consistency of TWO can't be achieved on a table that is local to a server.

**Comment by thibaultcha** [comment]
This should have been fixed in Kong 0.10.1 and the upcoming 0.10.2, thanks!

**Comment by jkumar-roambee** [comment]
nginx: [error] init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:200: [cassandra error] could not retrieve current migrations: [cassandra error] [Unavailable exception] Cannot achieve consistency level TWO
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:200: in function 'init'
	init_by_lua:3: in main chunk
